required/latex-lab/latex-lab-toc.dtx

```diff
@@ -308,30 +319,34 @@
 % \end{NOTE}
 %    \begin{macrocode}
            \prop_gput:Nee \g_@@_struct_ref_by_dest_prop
-             { \tag_get:n {struct_num} }{#3}
+             { \tag_get:n {struct_num} }{#4}
            \seq_gpush:Ne \g_@@_toc_stack_seq {{TOCI}\use:c{toclevel@#1}}
           \group_end:
        }
     }
   }
 %    \end{macrocode}
-% Now map it into the config point:
+% \end{macro}
+% Now map it into the socket:
```

Comment:
I think this should always (also earlier) say somehtingg like

Now define the tagging plug and assign it to the socket.